cmd/kube-scheduler/app/options/options.go

```diff
@@ -200,7 +201,7 @@ func (o *Options) ApplyTo(c *schedulerappconfig.Config) error {
 		}
 	}
 
-	return nil
+	return utilfeature.DefaultMutableFeatureGate.SetFromMap(o.ComponentConfig.FeatureGates)
```

Comment:
server.go seems like a location more consistent across k8s binaries for this call.

Reply:
I'm not so sure about enforcing a section for `FeatureGate` function calls, because it's the last stage of flag processing.

Reply:
Sg. I'm don't feel strong about it.